README.md

```diff
@@ -31,3 +34,13 @@
 
  > - cd  into the right directory and type `npm run build` or `yarn run build` in the terminal
```

Comment:
it should be NPM start/ yarn start and then if the user wants to build the app for production then user should use npm run build/yarn build.
**npm start command missing**